.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - run:
           name: Run Prettier over all files
           command: yarn prettier-check
```

Comment:
On that topic, checking prettier should simply be part of the `yarn lint` command, and the `lint` task. There's no functional use for it to be separate from sass-line, eslint.